boom! testing cargo-mod-0.1.4 against stable for beta
boom! running: cargo +stable build --frozen
boom! creating container for: cargo +stable build --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/beta/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/beta/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable build --frozen cargobomb`
blam! e2844559cfb76a680d63e1125e029502a5974921818648598f0e548447034145
boom! running `docker start -a e2844559cfb76a680d63e1125e029502a5974921818648598f0e548447034145`
kablam!    Compiling cargo-mod v0.1.4 (file:///source)
kablam!     Finished debug [unoptimized + debuginfo] target(s) in 1.51 secs
boom! running `docker rm -f e2844559cfb76a680d63e1125e029502a5974921818648598f0e548447034145`
blam! e2844559cfb76a680d63e1125e029502a5974921818648598f0e548447034145
boom! running: cargo +stable test --frozen --no-run
boom! creating container for: cargo +stable test --frozen --no-run
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/beta/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/beta/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable test --frozen --no-run cargobomb`
blam! 47311f03dcbeec6abbf215b721831a27481adf6a26432c435b8f23b7898a283a
boom! running `docker start -a 47311f03dcbeec6abbf215b721831a27481adf6a26432c435b8f23b7898a283a`
kablam!    Compiling cargo-mod v0.1.4 (file:///source)
kablam!     Finished debug [unoptimized + debuginfo] target(s) in 2.16 secs
boom! running `docker rm -f 47311f03dcbeec6abbf215b721831a27481adf6a26432c435b8f23b7898a283a`
blam! 47311f03dcbeec6abbf215b721831a27481adf6a26432c435b8f23b7898a283a
boom! running: cargo +stable test --frozen
boom! creating container for: cargo +stable test --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/beta/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/beta/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable test --frozen cargobomb`
blam! 1ba099c6762dd569eafbf96b9a52ba6f5a99e2bdf309f60add55b06d0132a930
boom! running `docker start -a 1ba099c6762dd569eafbf96b9a52ba6f5a99e2bdf309f60add55b06d0132a930`
kablam!     Finished debug [unoptimized + debuginfo] target(s) in 0.0 secs
kablam!      Running /target/debug/deps/cargo_mod-3c2d624a27e01f99
blam! 
blam! running 0 tests
blam! 
blam! test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured
blam! 
kablam!      Running /target/debug/deps/generate_modules-7047eae507ee53b2
blam! 
blam! running 5 tests
blam! test generate_module_inside_subdir ... FAILED
blam! test generate_multiple_modules ... FAILED
blam! test generate_private_module ... FAILED
blam! test generate_single_file_module ... FAILED
blam! test generate_single_folder_module ... FAILED
blam! 
blam! failures:
blam! 
blam! ---- generate_module_inside_subdir stdout ----
blam! 	thread 'generate_module_inside_subdir' panicked at 'failed to execute process: No such file or directory (os error 2)', tests/generate_modules.rs:129
blam! note: Run with `RUST_BACKTRACE=1` for a backtrace.
blam! 
blam! ---- generate_multiple_modules stdout ----
blam! 	thread 'generate_multiple_modules' panicked at 'failed to execute process: No such file or directory (os error 2)', tests/generate_modules.rs:70
blam! 
blam! ---- generate_private_module stdout ----
blam! 	thread 'generate_private_module' panicked at 'failed to execute process: No such file or directory (os error 2)', tests/generate_modules.rs:104
blam! 
blam! ---- generate_single_file_module stdout ----
blam! 	thread 'generate_single_file_module' panicked at 'failed to execute process: No such file or directory (os error 2)', tests/generate_modules.rs:81
blam! 
blam! ---- generate_single_folder_module stdout ----
blam! 	thread 'generate_single_folder_module' panicked at 'failed to execute process: No such file or directory (os error 2)', tests/generate_modules.rs:92
blam! 
blam! 
blam! failures:
blam!     generate_module_inside_subdir
blam!     generate_multiple_modules
blam!     generate_private_module
blam!     generate_single_file_module
blam!     generate_single_folder_module
blam! 
blam! test result: FAILED. 0 passed; 5 failed; 0 ignored; 0 measured
blam! 
kablam! error: test failed
boom! running `docker rm -f 1ba099c6762dd569eafbf96b9a52ba6f5a99e2bdf309f60add55b06d0132a930`
blam! 1ba099c6762dd569eafbf96b9a52ba6f5a99e2bdf309f60add55b06d0132a930
